### PR TITLE
Laboratory work #1, Alyona Gosteva - 21FPL2

### PR DESCRIPTION
**Pull request by Gostaa**

### CODE REVIEW

File: lab_1_keywords_tfidf/main.py [hunk 1]
```diff
@@ -93,16 +62,13 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
...
-    for token in tokens:
-        if token not in stop_words:
-            my_tokens.append(token)
-    return my_tokens
+
+    return [word for word in tokens if word not in stop_words]
 
 
-def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
+def calculate_frequencies(text_without_stopwords: list[str]) -> Optional[dict[str, int]]:
```
Comment (artyomtugaryov) [inline lab_1_keywords_tfidf/main.py:71]: Why did you change the interface?

File: lab_1_keywords_tfidf/main.py [hunk 2]
```diff
@@ -176,16 +146,14 @@ def calculate_tfidf(term_freq: dict[str, float], idf: dict[str, float]) -> Optio
...
-    for word in term_freq.keys():
-        tfidf_dict[word] = term_freq[word] * idf.get(word, log(47))
-    return tfidf_dict
+
+    return {word: freq * idf.get(word, log(47)) for word, freq in term_freq.items()}
 
 
 def calculate_expected_frequency(
-    doc_freqs: dict[str, int], corpus_freqs: dict[str, int]
+        doc_freqs: dict[str, int], corpus_freqs: dict[str, int]
```
Comment (artyomtugaryov) [inline lab_1_keywords_tfidf/main.py:156]: revert

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_1_keywords_tfidf/main.py (hunk 2 above)
Comment (Gostaa) [reply lab_1_keywords_tfidf/main.py:156]: done

File: lab_1_keywords_tfidf/main.py (hunk 1 above)
Comment (Gostaa) [reply lab_1_keywords_tfidf/main.py:71]: That's because of refractor :(
Name 'tokens' seemed not really meaningful to me, I was not thinking of it as of changing the interface

Comment (Gostaa) [reply lab_1_keywords_tfidf/main.py:71]: changed
